modules/lib.lua

```diff
@@ -235,6 +236,7 @@ if isServer then
         local warp = params.warp
         local ped = type(warp) == 'number' and warp or (sourceType == 'number' and warp and source or nil)
         local props = params.props
+        local bucket = params.bucket or nil
```

Comment:
why are we doing a "or nil"?

its either params.bucker or nil anyway, no reason to specify it

Reply:
```suggestion
        local bucket = params.bucket
```
because reasons